spec/iex/client_spec.rb

```diff
@@ -135,4 +135,36 @@
       end
     end
   end
+
+  context '#search' do
```

Comment:
tests for this endpoint were added to `client_spect.rb` because other endpoints have associated resources, where `#search` (eventually, according to the documentation) returns any type of resource.

Reply:
Make a directory, `search` and create `search/quote_spec.rb` and such? For the general use case of errors make a `search_spec.rb`.